APC updates for signin logs docs (#18596)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/authenticationappdevicedetails.md

---
title: "authenticationAppDeviceDetails resource type"

> description: "Provides details about the app and device used during an Azure AD authentication step."

Reviewer: ```suggestion
description: "Provides details about the app and device used during the Azure AD authentication step."
```

author: "besiler"
ms.localizationpriority: medium
ms.prod: "identity-and-access-reports"
doc_type: resourcePageType
---

# authenticationAppDeviceDetails resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Provides details about the authenticator app and device used during an authentication step. 

Reviewer: ```suggestion
Provides details about the app and device used during an Azure AD authentication step.
```

Author: Recommend keeping "authenticator app" as it's not the client app, but the authenticator app that we show details for here (the client app is in another part of the signin event).

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |appVersion|String|The version of the client app used during the authentication step.|

Reviewer: ```suggestion
|appVersion|String|The version of the client authentication app used during the authentication step.|
```

> |appVersion|String|Version of the client app used used during the authentication step.|

Reviewer: ```suggestion
|appVersion|String|The version of the client app used during the authentication step.|
```

> |clientApp|String|The name of the client app used during the authentication step.|

Reviewer: ```suggestion
|clientApp|String|The name of the authentication app used during the authentication step.|
```

> |clientApp|String|The name of the client app used used during the authentication step.|

Reviewer: ```suggestion
|clientApp|String|The name of the client app used during the authentication step.|
```

> |deviceId|String|ID of the device the used during the authentication step.|

Reviewer: ```suggestion
|deviceId|String|ID of the device used during the authentication step.|
```

> |operatingSystem|String|The type of operating system running on the deviced used for the authentication step.|

Reviewer: ```suggestion
|operatingSystem|String|The operating system running on the device used for the authentication step.|
```
